app/api/helpers/db.py

```diff
@@ -64,7 +69,12 @@ def get_or_create(model, **kwargs):
         return instance, was_created
 
 
-def get_count(q):
-    count_q = q.statement.with_only_columns([func.count()]).order_by(None)
-    count = q.session.execute(count_q).scalar()
+def get_count(query):
+    """
+    Counts how many items there are in a database object
```

Comment:
change `Counts how many items there are in a database object` to `Counts how many records are there in a database table/model`